tools/pubsys/src/aws/ssm/ssm.rs

```diff
@@ -220,7 +219,10 @@ pub(crate) async fn set_parameters(
         // need the region again here.)
         let mut throttled_streams = Vec::new();
         for (_region, request_list) in regional_requests {
-            throttled_streams.push(throttle(request_interval, stream::iter(request_list)));
+            throttled_streams.push(Box::pin(tokio_stream::StreamExt::throttle(
```

Comment:
Nice job figuring out pinning.

nit: can we `use` `throttle` like we used to?

Reply:
I tried again, and we can't. The best we could do is this:

```rust
use tokio_stream::StreamExt as TokioStreamExt;
```

```rust
        for (_region, request_list) in regional_requests {
            throttled_streams.push(Box::pin(TokioStreamExt::throttle(
                stream::iter(request_list),
                request_interval,
            )));
        }
```

I think what I have is better:

```rust
        for (_region, request_list) in regional_requests {
            throttled_streams.push(Box::pin(tokio_stream::StreamExt::throttle(
                stream::iter(request_list),
                request_interval,
            )));
        }
```